src/api/los/ApplicationDocumentsApi.ts

```diff
@@ -114,7 +114,7 @@ export default class ApplicationDocumentsApi extends BaseSystemApi<ApplicationDo
     return this.apiClient.makeCall<ApplicationDocument>(`/${this.path}`, 'POST', formData, { contentType: null });
   }
 
-  public createMany(applicationId: string, params: CreateManyApplicationDocumentParams): Promise<void> {
+  public createMany(applicationId: string, params: CreateManyApplicationDocumentParams, taskId?: string): Promise<void> {
```

Comment:
Pass taskId to params and forget to update changelog

Reply:
fixed
regarding changelog - I guess it should be updated once the package will be released to cloud to provide correct date, wdyt?

Reply:
Yup it's possible to do it in that way.